### DISCUSSION TITLE
Drag and Drop Menu Builder

### DISCUSSION THREAD

**Discussion by K-H-Rayhan** [clauderic/dnd-kit#1471] [Show and tell]
## React DND Menu Builder

For those seeking a menu builder for your dashboard, similar to the WordPress menu builder, I’m excited to introduce my drag-and-drop menu builder, developed using the dnd-kit. I would greatly appreciate your support—please leave a star if you find it helpful.

Source Code Link: https://github.com/K-H-Rayhan/react-dnd-menu-builder

https://github.com/user-attachments/assets/9fd97051-3c60-4d26-beef-aa01c618de06

**Comment by dogakorkmaz35** [comment]
I'll check this out!
